examples/a10_cgn_nat64/a10_cgnv6_nat64_alg_esp.yml

```diff
@@ -0,0 +1,13 @@
+- name: a10_cgnv6_nat64_alg_esp example playbook
+  connection: local
+  hosts: localhost
+  tasks:
+  - name: Create a10_cgnv6_nat64_alg_esp
+    a10_cgnv6_nat64_alg_esp:
+      a10_host: "10.43.12.137"
+      a10_username: "admin"
+      a10_password: "a10"
+      a10_port: "443"
+      a10_protocol: "https"
+      partition: "MyPartition"
```

Comment:
no sure if need  partition: "MyPartition" , because in most cases CGN is configured in a default, "shared" partition.